libs/anglify/src/modules/dialog/dialog.component.scss

```diff
@@ -5,6 +5,7 @@
   position: relative;
   display: flex;
   width: auto;
+  min-width: 240px;
```

Comment:
The dialog should have no size limit. This should be left to the developer

Reply:
https://material.io/components/dialogs have a min-width. I will make sure to set it as a variable then.

Reply:
<img width="329" alt="image" src="https://user-images.githubusercontent.com/20760160/174866007-f27125a5-c1bd-4c3b-b8c0-8f40e3d95034.png">
Not implementing the max-height though, but the width makes sense so it's not too squished when there is little text. This can now be adjusted with the css variable if you want that.